Tensile/Tests/extended/direct_to_vgpr/dtv_dgemm.yaml

```diff
@@ -77,8 +77,8 @@ BenchmarkProblems:
       BenchmarkFinalParameters:
         - ProblemSizes:
           #- Exact: [ M, N, B, K, ldc, ldc, lda, ldb ]
-          - Exact: [1024, 1024, 1, 512]
-          - Exact: [1022, 1022, 1, 511]
+          #- Exact: [1024, 1024, 1, 512]
+          - Exact: [1023, 1022, 1, 511]
```

Comment:
Did you mean either 1023x1023 or 1022x1022 here? Are the test sizes here checking for different things?

Reply:
N = 1022 is because I use AssertFree1 = [1,2]